Javascript/jq-test.js

```diff
@@ -1,69 +1,17 @@
-$( document ).ready(function() {
-    console.log( "ready!" );
-
+$(function() {
     $('#settings').click(function(){
       $('.bkg-menu').removeClass("hide");
```

Comment:
I've combined the individual `$('#settings').click(function() {` blocks into one, as this produces the same effect